metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -986,6 +988,27 @@ private UpdateAspectResult patchAspect(final Patch patch, final SystemMetadata s
     return result;
   }
 
+  public String batchApplyRetention(Integer start, Integer count, Integer attemptWithVersion, String aspectName,
+                                    String urn) {
+    BulkApplyRetentionArgs args = new BulkApplyRetentionArgs();
+    if (start == null) {
+      start = 0;
+    }
+    args.start = start;
+    if (count == null) {
+      count = 100;
+    }
+    args.count = count;
+    if (attemptWithVersion == null) {
+      attemptWithVersion = 21;
```

Comment:
Where does this value come from? What is the reason for this default?

Reply:
From `maxVersions: 20` in `/boot/retention.yml` 20 max versions + 1 default to 0.